File: Livro-FundamentosDaProgramacaoDeComputadores/Capitulo3/C/Exercicio2.c
//2. Faça um programa que receba três notas, calcule e mostre a média aritmética.

#include <stdio.h>

int main()
{
    float N1, N2, N3, Media = 0;

    printf("Digite os valore das tres notas: ");
        scanf("%f %f %f", &N1, &N2, &N3);
    
    Media = (N1 + N2 + N3) / 3;

    printf("\nA media das notas eh: %.1f", N1, N2, N3, Media);

    return 0;
}

File: Livro-FundamentosDaProgramacaoDeComputadores/Capitulo3/Java/Exercicio2.java
/*2. Faça um programa que receba três notas, calcule e mostre a média aritmética.*/

import java.util.Scanner;

public class Exercicio2 {
    public static void main(String[] args){

        Scanner scan = new Scanner(System.in);

        System.out.print("Digite a primeira nota: ");
        double nota1 = scan.nextDouble();
        System.out.print("Digite a segunda nota: ");
        double nota2 = scan.nextDouble();
        System.out.print("Digite a terceira nota: ");
        double nota3 = scan.nextDouble();

        double mediaAritmetica = (nota1 + nota2 + nota3) / 3;

        System.out.println("A media aritmética das notas é " + mediaAritmetica);

        scan.close();
    }
}